src/modules/functionality.js

```diff
@@ -0,0 +1,103 @@
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+
+// Elements
+
+const table = document.getElementById('table-body');
+const form = document.getElementById('toDoForm');
+const addToList = document.getElementById('addToList');
+
+// Add To Do
+
+function addToDo() {
+  const task = addToList.value;
+
+  todos.push({
+    description: task,
+    completed: false,
+    index: todos.length,
+  });
+  addToList.value = '';
```

Comment:
- [ ] _Index_ property of added item does not match new array length. For instance, if a 5th item is added to the list, the index of that item should equal to 5. Kindly ensure to improve upon the same.

**Mismatch in index property and array length:**
![image](https://user-images.githubusercontent.com/12745474/207928101-83894bfc-9a40-4346-89af-5eb1742edb07.png)